pkg/dssproto/rid.pb.go

```diff
@@ -0,0 +1,4852 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.21.0
+// 	protoc        v3.6.1
+// source: pkg/dssproto/rid.proto
+
+package dssproto
```

Comment:
Should we change this to dssproto_rid or something like that to preemptively make it easier to deal with object naming conflicts between different APIs? (perhaps something for a later PR)

Reply:
Agreed. I would also consider naming clashes a code smell and investigate case by case.